### DISCUSSION TITLE
Hide folder names starting with a "." (dot simbol) in the subfolder list

### DISCUSSION THREAD

**Discussion by hakaserver** [BlafKing/sd-civitai-browser-plus#105] [Ideas]
Currently the Subfolder list shows every folder available in the models folder, including folders that start with a dot "." in the name. That behaviour differs from the default behaviour of a1111's extra networks tabs, in which folders beginning with a dot in their name are treated as hidden folders (they are searcheable, but do not show as buttons in the UI for organization purposes).

My suggestion would be to have an option to hide such folders from the Subfolder list for people who, like me, use such hidden folders to hide long folder paths, to better organize the UI. Having them also hidden in the Subfolder list would make it easier to select the non hidden, more frequently used folders when downloading new models.

**Comment by BlafKing** [comment]
Nice suggestion!

This option has now been added to the settings in v2.0! :)

**Reply by hakaserver** [reply]
Awesome! Thanks for the addition!
